web/packages/extension/assets/_locales/en/messages.json

```diff
@@ -11,6 +11,9 @@
     "settings_show_swf_download": {
         "message": "Show SWF download in context menu"
     },
+    "settings_autostart": {
+        "message": "Disable splash screen and autoplay (click to unmute)"
```

Comment:
It's ambiguous whether this is "disable(splash screen) AND disable(autoplay)" or "disable(splash screen) AND do(autoplay)". At least to me.

Reply:
And I don't know what to do with the "(click to unmute)" part.
I mean, intuitively, I can guess what this is supposed to do, but to a newcomer, it might be difficult to parse.

Reply:
The original phrasing of this option was "Disable splash screen and autoplay content on mute (click to unmute)". I changed it because I found it redundant and wordy (taking up several lines) but I can switch back to that to remove ambiguity.

Reply:
Or maybe "Autoplay content on mute and disable splash screen (click content to unmute)"

Reply:
How about something like "Play automatically on click, without splash screen" or "Start on click, with no splash screen"?

Reply:
But it plays automatically without click. It just needs clicks on the content for it to unmute.

Reply:
Oh, then I misunderstood...
In that case, how about something along the lines of "Play automatically without splash screen (then click to unmute)"?

Reply:
In the future we might want to add tooltips to provide further explanation for this and several other options that are difficult to describe succinctly. For now, let's not get too hung up on the wording.

Reply:
I agree we shouldn't get too caught up with bike-shedding. But I do think torokati's last suggestion is probably worded better, so I went with it. It's a bit long, but still acceptable I suppose.

I can remove this from the popup panel if we feel this is more of a power user feature. Just to show the current length of the text, here are screenshots of the options page and popup panel. If this were removed from the popup panel and I switched from "Play automatically" back to "autoplay", it would fit on one line on the options page:
![Screenshot from 2023-04-11 09-23-30](https://user-images.githubusercontent.com/29206584/231176777-29a4df74-5228-4c73-aaf2-b3ac34c397e2.png)
![Screenshot from 2023-04-11 09-23-16](https://user-images.githubusercontent.com/29206584/231176786-e7f958a6-031c-42aa-a71c-854e90db3c44.png)

Reply:
Looks fine to me, and I think it's good to have on the popup because it's a widely useful feature. Autoplay is one of the features that one ripoff extension was making people pay for, after all 😉

Reply:
Sounds good

Reply:
BTW if the width of the popup was increased a little bit, even this long option weren't broken into 3 lines, so it would look a lot less crowded. But this is a separate design thing...